projects/cds/src/merchandising/cms-components/merchandising-carousel/merchandising-carousel.component.html

```diff
@@ -7,6 +7,7 @@
     [cxAttributesNamePrefix]="'data-cx-merchandising-carousel'"
   ></div>
```

Comment:
do we need this class="data-cx-merchandising-product"

not visible in this file

Reply:
I am not 100% sure, perhaps so there is a simple unique selector using the id and then a collective selector by using this class? It is how the current carousel has done it, so I would be hesitant to change it to much in case the profile tag scraping relies on it

Reply:
We will remove it in the future styling PR